backend/tests/services/rag/test_remote_gateway.py

```
@@ -687,3 +677,143 @@ def test_gateway_factory_supports_per_operation_overrides(monkeypatch) -> None:
+                    model_name="test",
+                    model_namespace="default",
+                    resolved_config={"protocol": "openai", "model_id": "test-model"},
+                ),
+                retrieval_config=RuntimeRetrievalConfig(),
+            )
+        ],
+    )
+
+    await gateway.query(spec)
+
+    args, kwargs = post_mock.await_args
```

Ruff (0.15.10)

[warning] Unpacked variable `args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)